.github/workflows/benchmark-nightly.yml

```diff
@@ -24,6 +24,8 @@ env:
   ARM_SUBSCRIPTION_ID: ${{ secrets.BENCHMARK_ARM_SUBSCRIPTION_ID }}
   ARM_TENANT_ID: ${{ secrets.BENCHMARK_ARM_TENANT_ID }}
   K6_API_TOKEN: ${{ secrets.K6_API_TOKEN }}
+  K6_RESULT_WEBHOOK_URL: ${{ secrets.K6_RESULT_WEBHOOK_URL }}
```

Comment:
Would it make sense to not use "K6" in the names for these env vars? After all, the fact that we're using k6 is just an implementation detail, isn't it?

Reply:
Good point. I'll rename